cpu/Makefile.include.cortexm_common

```diff
@@ -70,6 +120,14 @@ endif
 # CPU depends on the cortex-m common module, so include it:
 include $(RIOTCPU)/cortexm_common/Makefile.include
 
+# Import all toolchain settings
+include $(RIOTBOARD)/Makefile.include.$(TOOLCHAIN)
```

Comment:
I agree, although I would like to take it one step further and move them all into a separate makefile directory, see #4699